apps/src/aichat/redux/aichatRedux.ts

```diff
@@ -645,7 +572,6 @@ export const {
   setNewChatSession,
   setChatSessionId,
   clearChatMessages,
-  setIsWaitingForChatResponse,
```

Comment:
This is the only intended diff that's not just moving things from one place to another -- my editor is telling me this is unused, and I didn't see any usages.